### PR TITLE
[Relay] Add an endpoint /v1/mobile/exist/:account

### PR DESCRIPTION
**Pull request by danial303065**

### CODE REVIEW

File: packages/relay/src/routers/ETCRouter.ts [hunk 1]
```diff
@@ -95,6 +95,12 @@ export class ETCRouter {
             [param("account").exists().trim().isEthereumAddress(), query("type").exists()],
             this.mobile_info.bind(this)
         );
+
+        this.app.get(
+            "/v1/mobile/exist/:account",
```
Comment (MichaelKim20) [inline packages/relay/src/routers/ETCRouter.ts]:
```suggestion
            "/v1/mobile/exists/:account",
```

File: packages/relay/src/routers/ETCRouter.ts [hunk 2]
```diff
@@ -221,4 +227,38 @@ export class ETCRouter {
             return res.status(200).json(msg);
         }
     }
+
+    /**
+     * GET /v1/mobile/exist
+     * @private
+     */
+    private async mobile_exist(req: express.Request, res: express.Response) {
+        logger.http(`GET /v1/mobile/exist ${req.ip}:${JSON.stringify(req.params)}`);
```
Comment (MichaelKim20) [inline packages/relay/src/routers/ETCRouter.ts]:
```suggestion
        logger.http(`GET /v1/mobile/exists ${req.ip}:${JSON.stringify(req.params)}`);
```

File: packages/relay/src/routers/ETCRouter.ts [hunk 3]
```diff
@@ -221,4 +227,38 @@ export class ETCRouter {
             return res.status(200).json(msg);
         }
     }
+
+    /**
+     * GET /v1/mobile/exist
```
Comment (MichaelKim20) [inline packages/relay/src/routers/ETCRouter.ts]:
```suggestion
     * GET /v1/mobile/exists
```

File: packages/relay/src/routers/ETCRouter.ts [hunk 4]
```diff
@@ -221,4 +227,38 @@ export class ETCRouter {
...
+            return res.status(200).json(
+                this.makeResponseData(0, {
+                    account,
+                    type,
+                    exist,
+                })
+            );
+        } catch (error: any) {
+            const msg = ResponseMessage.getEVMErrorMessage(error);
+            logger.error(`GET /v1/mobile/exist : ${msg.error.message}`);
```
Comment (MichaelKim20) [inline packages/relay/src/routers/ETCRouter.ts]:
```suggestion
            logger.error(`GET /v1/mobile/exists : ${msg.error.message}`);
```

File: packages/relay/src/routers/ETCRouter.ts [hunk 5]
```diff
@@ -221,4 +227,38 @@ export class ETCRouter {
...
+        const errors = validationResult(req);
+        if (!errors.isEmpty()) {
+            return res.status(200).json(ResponseMessage.getErrorMessage("2001", { validation: errors.array() }));
+        }
+
+        try {
+            const account: string = String(req.params.account).trim();
+            const type: number = Number(req.query.type);
+            const mobileData = await this.storage.getMobile(account, type);
+            const exist = mobileData !== undefined && mobileData.account.toLowerCase() === account.toLowerCase();
```
Comment (MichaelKim20) [inline packages/relay/src/routers/ETCRouter.ts]:
```suggestion
            const exists = mobileData !== undefined && mobileData.account.toLowerCase() === account.toLowerCase();
```

File: packages/relay/src/routers/ETCRouter.ts [hunk 6]
```diff
@@ -221,4 +227,38 @@ export class ETCRouter {
...
+
+            this.metrics.add("success", 1);
+            return res.status(200).json(
+                this.makeResponseData(0, {
+                    account,
+                    type,
+                    exist,
```
Comment (MichaelKim20) [inline packages/relay/src/routers/ETCRouter.ts]:
```suggestion
                    exists,
```

**Review by MichaelKim20 — CHANGES_REQUESTED** [review]
Check please

**Comment by danial303065** [comment]
@MichaelKim20 Review please

**Review by MichaelKim20 — APPROVED** [review]
LGTM

**Pull request merged**